library/sb-1.4.1.js

```diff
@@ -575,6 +576,35 @@ Spacebrew.Client.prototype._onClose = function() {
 };
 
 /**
+* Called on WebSocket error
+* @private
+* @memberOf Spacebrew.Client
+*/
+Spacebrew.Client.prototype._onError = function(e) {
+ var self = this;
+ console.log("[_onError:Spacebrew] Spacebrew connection error");
+
+ this._isConnected = false;
+ if (this.admin.active) {
+	 this.admin.remoteAddress = undefined;
+ }
+
+ // if reconnect functionality is activated set interval timer if connection dies
+ if (this.reconnect && !this.reconnect_timer) {
+	 console.log("[_onError:Spacebrew] setting up reconnect timer");
+	 this.reconnect_timer = setInterval(function () {
+			 if (self.isConnected !== false) {
+				 self.connect();
+				 console.log("[reconnect:Spacebrew] attempting to reconnect to spacebrew");
+			 }
+		 }, 5000);
+ }
+
+
+ this.onError(e);
```

Comment:
is there a reason to not add a stub to the prototype for onError similar to the message handlers? (onBooleanMessage etc)

Reply:
No reason. Good catch. I also fixed the param JSDoc for _onError. Both changes on my latest master branch.